src/Symfony/Component/Mime/Tests/EmailTest.php

```diff
@@ -0,0 +1,385 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Mime\Tests;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Mime\Address;
+use Symfony\Component\Mime\Email;
+use Symfony\Component\Mime\NamedAddress;
+use Symfony\Component\Mime\Part\DataPart;
+use Symfony\Component\Mime\Part\Multipart\AlternativePart;
+use Symfony\Component\Mime\Part\Multipart\MixedPart;
+use Symfony\Component\Mime\Part\Multipart\RelatedPart;
+use Symfony\Component\Mime\Part\TextPart;
+
+class EmailTest extends TestCase
+{
+    public function testSubject()
+    {
+        $e = new Email();
+        $e->subject('Subject');
+        $this->assertEquals('Subject', $e->getSubject());
+    }
+
+    public function testDate()
+    {
+        $e = new Email();
+        $e->date($d = new \DateTimeImmutable());
+        $this->assertSame($d, $e->getDate());
+    }
+
+    public function testReturnPath()
+    {
+        $e = new Email();
+        $e->returnPath('fabien@symfony.com');
+        $this->assertEquals(new Address('fabien@symfony.com'), $e->getReturnPath());
+    }
+
+    public function testSender()
+    {
+        $e = new Email();
+        $e->sender('fabien@symfony.com');
+        $this->assertEquals(new Address('fabien@symfony.com'), $e->getSender());
+
+        $e->sender($fabien = new Address('fabien@symfony.com'));
+        $this->assertSame($fabien, $e->getSender());
+    }
+
+    public function testFrom()
+    {
+        $e = new Email();
+        $helene = new Address('helene@symfony.com');
+        $thomas = new NamedAddress('thomas@symfony.com', 'Thomas');
+        $caramel = new Address('caramel@symfony.com');
```

Comment:
Who is Caramel 😸 ?

Reply:
Who ever it is, I'm pretty sure they are sweet 😏

Reply:
My cat, correct :)